src/components/DetailsPage/DetailsPage.js

```diff
@@ -14,6 +15,10 @@ function DetailsPage({
   categories,
   price,
 }) {
+  useEffect(() => {
+    window.scrollTo(0, 0)
+  }, [])
```

Comment:
I'm not super familiar with your apps architecture, but this seems like something you wouldn't need to do on every page. Maybe you could try moving this to your App component and add your pathname (from useLocation) as a dependency.

See here: https://reactrouter.com/web/guides/scroll-restoration

Reply:
Thanks for the review and the link, I think I'll create a component for the scrolling that I'll reference in the other components 👍